multipy/runtime/interpreter/plugin_torch.cpp

```diff
@@ -15,10 +15,11 @@ class TorchConverter : public Converter {
   }
 
   ~TorchConverter() override {
+    deregisterConverter(this);
     // GetPythonFramesFunction depends on the current python so we need to tear
     // it down.
     // https://www.internalfb.com/code/fbsource/[ead2dd7fd4a6]/fbcode/caffe2/torch/csrc/lazy/python/init.cpp?lines=293
-    ::torch::lazy::GetPythonFramesFunction() = nullptr;
+    //::torch::lazy::GetPythonFramesFunction() = nullptr;
```

Comment:
nit: remove. Do we not need this anymore?

Reply:
Still need some more testing -- I was getting destruction order issues with this since libtorch_python was having issues destructing in the right order. I think it's fine now but I haven't seen any issues with this so it's likely fine to remove